src/react-apps/ux-editor/src/components/config/ApiConfigurationComponent.tsx

```diff
@@ -307,7 +308,7 @@ class ApiConfiguration extends React.Component<IApiConfigurationProps, IApiConfi
             checked={this.state.apiType === 'externalapi'}
           />
           <label className='custom-control-label pl-3 a-radioButtons-title'>
-            {'External API'}
+            {this.props.language ? this.props.language['ux-editor'].modal_configure_api_extermnal_api : 'External API'}
```

Comment:
Why can't this be written this.props.language.ux-editor?

Reply:
You can't use - in referencing. This gives an ts errors saying "missing whitespace and Can't find editor"

Reply:
In that case maybe we should rename it to uxEditor? and write in the documentation that sections should not contian "-" or something